core/src/main/java/org/mskcc/cbio/oncokb/bo/DrugGroupBo.java

```diff
@@ -0,0 +1,20 @@
+package org.mskcc.cbio.oncokb.bo;
+
+import org.mskcc.cbio.oncokb.model.Drug;
+import org.mskcc.cbio.oncokb.model.DrugGroup;
+
+import java.util.List;
+
+public interface DrugGroupBo extends GenericBo<DrugGroup> {
+    /**
+     * @param groupName
+     * @return
+     */
+    DrugGroup findGroupByName(String groupName);
+
+    /**
+     * @param drug
+     * @return
+     */
+    List<DrugGroup> findGroupsByDrug(Drug drug);
```

Comment:
Do we also need `findDrugsInGroup` in order to match trials?

Reply:
I don't think so. The drug table should cover all drugs. Unless they may mention anything like `MEK inhibitors`?

Reply:
Right, we need that as well. For additional endpoints, I can add them when I work on the frontend features.